README.md

```diff
@@ -33,3 +33,40 @@ new EditorView({
 });
 
 ```
+
+### Options
+
+You can provide an options object to `indentationMarkers()` with the following
+optional properties:
+
+- `activeBlockMarkers`
```

Comment:
naming nit: I prefer to start options with verbs if applicable. also think `markers` is redundant here. in this case, can we call this `highlightActiveBlock`?

Reply:
Makes sense. Also, trivial to change, yay TypeScript.